src/Handler/EasyHandle.php

```diff
@@ -74,7 +74,7 @@ public function createResponse()
 
         // Attach a response to the easy handle with the parsed headers.
         $this->response = new Response(
-            $startLine[1],
+            isset($startLine[1]) ? (integer) $startLine[1] : 200,
```

Comment:
Should be `(int)` rather than `(integer)`.

Reply:
Also, rather than 200, doesn't Guzzle have an http ok constant that could be used?

Reply:
Fixed to (int) - about const - I did not found any statuses constants.
But I don't sure is better to stay 200 status or maybe we can have some better variants

Reply:
No constants AFAIK, just this list: https://github.com/guzzle/psr7/blob/master/src/Response.php#L15